### DISCUSSION TITLE
PLS UPDATE DIEP IO MOBILE !!!

### DISCUSSION THREAD

**Discussion by ssjj-maker** [benawad/dogehouse#1960] [Ideas]
pls update diep mobile it haven't been updated for a long time.. people are starting to loosing interest in it WE ARE ALL WAITING FOR A UPDATE.. I got an I a idea on what update you can do, pls try to make new tanks or a new sandbox in diep mobile :) Thank you so much for reading this.. Diep forever <3

**Comment by amitojsingh366** [comment]
the wrong repo you got there my friend...
